### PR TITLE
feat: add SlimeFrame fast foundry

### PR DESCRIPTION
**Pull request by VoperAD**
## Description

It basically adds support for SlimeFrame (new dependency in pom.xml) and a new fast machine for the multiblock "Foundry" from SlimeFrame.

## Proposed changes

- New fast machine: Fast SlimeFrame Foundry

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

No documentation was added, I just followed the pattern used in the creation of the fast infinity workbench.

### CODE REVIEW

File: src/main/java/net/guizhanss/fastmachines/setup/Items.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import net.guizhanss.fastmachines.items.machines.slimeframe.FastSlimeFrameFoundry;
```
Comment (ybw0014) [inline src/main/java/net/guizhanss/fastmachines/setup/Items.java:6]: import order doesnt seem right here

**Review by ybw0014 — CHANGES_REQUESTED** [review]
Thank you for your contribution! One small change is needed here.

**Comment by VoperAD** [comment]
Done! I have just fixed it.

**Review by ybw0014 — APPROVED** [review]
lgtm

**Pull request merged**